pkg/pillar/cmd/volumemgr/updatestatus.go

```diff
@@ -227,6 +227,22 @@ func doUpdateContentTree(ctx *volumemgrContext, status *types.ContentTreeStatus)
 				blobErrorEntities = append(blobErrorEntities, &types.ErrorEntity{EntityID: blob.Sha256, EntityType: types.ErrorEntityContentBlob})
 
 				leftToProcess = true
+				if blob.RetryCount < blobDownloadMaxRetries {
```

Comment:
I think this check should be guarded by a check if status.IsErrorSource(types.VerifyImageStatus{}) to avoid any false triggers.